bench/Main.hs

```diff
@@ -3,9 +3,10 @@
 
 module Main where
 
+-------------------------------------------------------------------------------
+
```

Comment:
Yeah I'm definitely open to alternatives here; I insert these block comments because I have a really hard time navigating source files without a visual demarcation establishing the boundaries between logical sections (if that makes sense).